tracim/lib/utils/request.py

```
@@ -214,6 +214,8 @@ def _get_current_comment(
         comment_id = ''
         try:
             if 'comment_id' in request.matchdict:
+                if not request.matchdict['comment_id'].isdecimal():
+                    raise ContentNotFoundInTracimRequest('comment_id is not a correct integer')  # nopep8
```

Do not raise a `NotFound` exception if the variable is found but invalid. Raise an `Invalid` exception

---

note : for every check `isdecimal()` you should replace it with a `isint()` and raise a `InvalidContentId`

---

isint() is not a valid python method as far i know, i choose isdecimal() method from std lib of string because it the more strict method i found to select this kind of id.
Not sure what i should do.

---

This should be checked in #93, not here.